code/src/layouts/header/messages/uk.json

```diff
@@ -2,5 +2,4 @@
   "header.searchInputPlaceholder": "Пошук...",
   "header.searchInputNoResults": "Результатів не знайдено",
   "header.searchInputResults": "Результати пошуку:"
-
 }
```

Comment:
fix end line

Reply:
Thank you but I don't have this line here )
![image](https://github.com/user-attachments/assets/907e0b66-d04e-49dd-812a-2d1802536da5)

Reply:
Fixed